components/Diagram.js

```diff
@@ -84,7 +85,7 @@ const Diagram = ({diagram, coins}) => {
       ],
       labels: labels,
     });
-  }, [diagram]);
+  }, [diagram, coins]);
 
   /*   const textCenter = {
```

Comment:
Please delete code you don't use anymore.

Reply:
done!